### PR TITLE
Implemented ability to read all publishers

### PR DESCRIPTION
**Pull request by justin-obrien-smoothstack**
Books aren't yet included in the publisher objects.

### CODE REVIEW

File: controller/publisherCrudController.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const router = require('express').Router(),
+    service = require('../service/publisherCrudService');
+
+router.post('', () => {
+
+});
+
+router.get('', () => {
```
Comment (trevorhuis-smoothstack) [inline controller/publisherCrudController.js:10]: I'm guessing these are coming later?

File: service/publisherCrudService.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    const responseAttributes = {};
+    return new Promise((resolve, reject) => {
+        dao.read().then(result => {
+            responseAttributes.status = 200
+            responseAttributes.message = result.length === 0 ?
+                'There are no publishers in the database.' : result;
+            resolve(responseAttributes);
+        }).catch(error => {
+            responseAttributes.status = 500;
+            responseAttributes.message = 'There was an error while attempting to ' +
```
Comment (trevorhuis-smoothstack) [inline service/publisherCrudService.js:23]: Is prettier making this '+' to break the line? Or is there another reason for now keeping the strings together?

**Review by trevorhuis-smoothstack — APPROVED** [review]
looks good, don't forget to make some with callbacks!

File: service/publisherCrudService.js (hunk 2 above)
Comment (justin-obrien-smoothstack) [reply service/publisherCrudService.js:23]: I don't have Prettier installed yet. That was just me trying to keep lines to a maximum length of 80 characters.

**Pull request merged**